aten/src/TH/THMemoryFile.c

```diff
@@ -1,6 +1,7 @@
 #include "THMemoryFile.h"
 #include "THFilePrivate.h"
 #include "stdint.h"
+#include <sys/types.h>
```

Comment:
@peterjc123 this has to be wrapped in `#ifndef _WIN32`, right?

Reply:
yes, i think so.